### DISCUSSION TITLE
Allow members to create templates without being workspace admins

### DISCUSSION THREAD

**Discussion by amoralesc** [outline/outline#6202] [Ideas]
Currently, only workspace admins can manage templates. It would be great if members could also have a permission to manage templates. This permission could be a toggled feature located at `Preferences > Security > Behavior`, like the `Collection creation` permission. See attached picture below:

![template management proposed solution](https://github.com/outline/outline/assets/78283504/73fde9cf-0d9a-40ca-b918-afd828f74636)

It would be great if one could manage permissions on a user-per-user or group-per-group basis, but since there doesn't seem to be a feature like that to any other permission, a general permission for all members is a good enough solution.

The idea mostly comes from the fact that workspace admins are required to:
- Create other workspace admins when members want to create a template, which is a security vulnerability; or
- manually create and edit the template themselves. While this doesn't happen that often, it is still kind of annoying for admins

**Comment by tommoor** [comment]
Agreed, you shouldn't need to be an admin to manage templates.

**Reply by amoralesc** [reply]
Can we create an issue as a feature request for this?

**Reply by tommoor** [reply]
https://github.com/outline/outline/issues/6203
